### ISSUE TITLE
Table only accepts nested <Td> elements

### ISSUE THREAD

**Issue by okcoker** [glittershark/reactable#122]
Currently it seems as the Table only accepts `<Reactable.Td>` elements.

I am trying to customize the `<Reactable.Th>` columns to have the labels, along with some "unsafe" html. Something like this:

```
         <Table
                sortable={true}>
                <Thead>
                    <Tr>
                        <Th>header1</Th>
                        <Th>header2</Th>
                        <Th>header3</Th>
                        <Th>header4</Th>
                        <Th>header5</Th>
                    </Tr>
                </Thead>
                { body_rows }
            </Table>
```

This renders my table with the appropriate data, but under an empty row. Seems as though adding my own `Thead` and `Th` elements causes an empty row and is completely ignored by Reactable. `body_rows` is an array of `Tr` elements but as soon as I try to be semantically correct and add a `Tbody` around it, no table is rendered. If the only children are `Tr`'s things are fine but I want to be able to customize cell content. Some of my problems I think revolve around [this](https://github.com/glittershark/reactable/blob/master/src/reactable.jsx#L504). 

I'll see if I can fix some things because I really like this library so far, but I at least wanted to document this as a ticket first.

Thanks for listening.

**Comment by glittershark** [comment]
Yeah, there's a fair amount of (admittedly hackish) checking of types of child components - that line you linked is almost definitely the place to start.

If you work at it and get stuck, I always like pull requests with failing tests :smile:

**Comment by glittershark** [comment]
So it looks like React.addons.TestUtils has a [function][] to determine the type of a React component. Might be good to just directly copy+paste the source of that function (since we don't want to require people include the entire testutils into their page)

[function]: http://facebook.github.io/react/docs/test-utils.html#iselementoftype

**Comment by quatrix** [comment]
I'm trying to customize <Th> to have a checkbox, a 'select all' type of thing and having the same problem.
